api/server/server.go

```diff
@@ -59,18 +62,21 @@ type Server struct {
 	minerService      *miner.Service
 	slashingService   *slashing.Service
 
-	rpc *grpc.Server
+	grpcServer   *grpc.Server
+	grpcWebProxy *http.Server
 
 	closeLotus func()
 }
 
 // Config specifies server settings.
 type Config struct {
-	LotusAddress    ma.Multiaddr
-	LotusAuthToken  string
-	GrpcHostNetwork string
-	GrpcHostAddress string
-	RepoPath        string
+	LotusAddress        ma.Multiaddr
+	LotusAuthToken      string
+	GrpcHostNetwork     string
+	GrpcHostAddress     string
+	GrpcServerOpts      []grpc.ServerOption
```

Comment:
Bubbling these config options all the way up so was can easily add tls configuration at some point